common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -156,16 +160,16 @@ class AccountInfo extends React.Component<Props, State> {
                 onClick={this.toggleShowLongBalance}
               >
                 <UnitDisplay
-                  value={balance.wei}
+                  value={etherBalance}
```

Comment:
Unrelated code refactor while I was cleaning up the types in this component. My bad.

Reply:
👻